osquery/tables/events/linux/bpf_process_events.cpp

```diff
@@ -42,6 +42,11 @@ bool BPFProcessEventSubscriber::generateRow(
     return false;
   }
 
+  auto signed_exit_code = static_cast<std::int64_t>(event.bpf_header.exit_code);
+  if (signed_exit_code == -1LL) {
```

Comment:
I think you want to check for -2:

```python
>>> import ctypes
>>> k = -2
>>> ctypes.c_ulong(k).value
18446744073709551614L
```

Reply:
I mistakenly used the exit code documented in the glibc wrapper; we should instead check whether the returned value is in the valid range of a negative errno value